### PR TITLE
Browse chefs integration

### PR DESCRIPTION
**Pull request by smithpg**
Partial implementation of the front end for browsing chefs. When the page loads, a GET request is sent to the backend to retrieve all chefs. JSON response is fed as props to ChefCard component. 

Note: Filter by location is not yet functional

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -20,8 +21,9 @@ function App() {
         </Route>
         <Route path="/(signup||login)" component={AuthPage} />
         <Route path="/chef/:chef_id" component={ChefPage}/>
-        <Route path="/customer/:customer_id" component={CustomerPage}/>
-        <Route path="/browse" />
+
+        <Route path="/customer/:customer_id" />
```
Comment (aeciorc) [inline client/src/App.js]: Is this one missing the component prop?

File: client/src/hooks/useMockedResource.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { useState, useEffect, useContext } from "react";
+
+const mockData = require("../helpers/mockData.json");
```
Comment (aeciorc) [inline client/src/hooks/useMockedResource.js]: This file is not in the repo, so this line breaks the client

File: client/src/hooks/useToggle.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { useState } from "react";
```
Comment (aeciorc) [inline client/src/hooks/useToggle.js:1]: Great practice to reuse logic!

File: client/src/pages/AuthPage.js [hunk 4]
```diff
@@ -63,7 +63,7 @@ function AuthPage({ classes, ...rest }) {
 
   return (
     <PageContainer className="pageContainer">
-      <Navbar>
+      <Navbar transparent={true }>
```
Comment (aeciorc) [inline client/src/pages/AuthPage.js]: This is equivalent to: ` <Navbar transparent>` , it automatically takes a `true` value. Matter of choice though

File: server/bin/config/custom-environment-variables.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+    "jwtprivatekey": "chefsmenujwtprivatekey"
```
Comment (aeciorc) [inline server/bin/config/custom-environment-variables.json:2]: I think you don't want this file on the repo

**Review by aeciorc — CHANGES_REQUESTED** [review]

File: server/bin/config/custom-environment-variables.json (hunk 5 above)
Comment (smithpg) [reply server/bin/config/custom-environment-variables.json:2]: The value is the name of an env variable. This file basically just allows the config module to lookup the env variable so we can access non-sensitive config variables and sensitive environment variables through the same interface, i.e. config.get(propertyName).

File: client/src/pages/AuthPage.js (hunk 4 above)
Comment (smithpg) [reply client/src/pages/AuthPage.js]: I think briefer is better here, thanks for catching that

File: client/src/App.js (hunk 1 above)
Comment (smithpg) [reply client/src/App.js]: Fixed

File: client/src/hooks/useMockedResource.js (hunk 2 above)
Comment (smithpg) [reply client/src/hooks/useMockedResource.js]: Indeed. I'm not sure it makes sense to have a dedicated hook for mocking anyway, so I just deleted the file

File: server/bin/config/custom-environment-variables.json (hunk 5 above)
Comment (aeciorc) [reply server/bin/config/custom-environment-variables.json:2]: Oh ok, got it, I saw that and assumed it was the value of the key itself

**Pull request merged**